bitnami/kafka/templates/svc-external-access.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.externalAccess.enabled -}}
+  {{- $fullName := include "kafka.fullname" . }}
+  {{- $replicaCount := .Values.replicaCount | int }}
+  {{- $root := . }}
+
+{{- range $i, $e := until $replicaCount }}
+  {{- $targetPod := printf "%s-%d" (printf "%s" $fullName) $i }}
```

Comment:
Why do you use this indentation?

Reply:
no reason, just to see easily the `if` and `range` blocks